Cargo.toml

```diff
@@ -24,7 +24,7 @@ default-members = ["wgpu", "wgpu-hal", "wgpu-info"]
 [patch."https://github.com/zakarumych/gpu-alloc"]
 #gpu-alloc = { path = "../gpu-alloc/gpu-alloc" }
 
-[patch."https://github.com/grovesNL/glow"]
+[patch."https://github.com/caiiiycuk/glow"]
```

Comment:
we are blocked on this currently?

Reply:
I don't know why maintainer of glow is not responding to my PR. I think it's better to wait until he merges it...